cypress/e2e/ReplacementAttorneysHW.feature

```diff
@@ -23,7 +23,7 @@ Feature: Add Replacement Attorneys to a Health and Welfare LPA
         And I can find "postcode-lookup"
         And I can find "name-title" with 8 options
         When I select "Ms" on "name-title"
-        And I force fill out
+        And I quickly fill out
```

Comment:
What does 'quickly' really mean here?  wonder if this could be named a bit more precisely?

Reply:
I added another method to fill a form in which doesn't use `type()`. This just speeds up filling forms. It can't be used everywhere as it doesn't trigger change events the same way (so can't be used to test the client-side validation).

Reply:
If you've got a suggestion for a better name, I'm happy to change it.

Reply:
Or alternatively, should we make everything do it the "quick" way and take out the word "quickly" ?  At the moment some of the filling out is done this way and some the conventional way which is inconsistent and could makes tests behave weirdly maybe?  But this could be another story and PR of course.  :). What do you think?

Reply:
The problem is that you can't use the "quickly" technique for all of the tests. Where the user interface reacts to changes in an input (e.g. where we prompt the user about how old/young the attorney is when they edit their birthdate), the "quickly" method doesn't work: in effect, it just sets the value attribute on the input, which doesn't trigger the change events on the input. So we can either keep the "quickly" approach (i.e. fill form inputs en masse where the JS event triggering doesn't matter), which as you rightly say is inconsistent; or revert to using the "force" approach everywhere (which potentially slows things down).